kani-driver/src/args.rs

```diff
@@ -4,14 +4,14 @@
 #[cfg(feature = "unsound_experiments")]
 use crate::unsound_experiments::UnsoundExperimentArgs;
 
-use clap::{error::Error, error::ErrorKind, CommandFactory, Parser, ValueEnum};
```

Comment:
Inessential, but I (1) noticed we were deriving `Parser` for everything, but not everything was actually a parser and there were more specific options (2) the clap documentation often seems to use `clap::Parser` instead of importing it, so it seemed conventional, but really: (3) removing it let me easily audit every use of it in the file.

Reply:
It is odd, but not a blocker.